### ISSUE TITLE
What is readonly parameter functionality 

### ISSUE THREAD

**Issue by MuhammadTalhaAfzal** [juank11memphis/angular-jk-rating-stars#21]
Can anybody tell me what readonly parameter do? and how can i remove reset button or restrict users, so that they can't change stars?

**Comment by msg2santoshsagar** [comment]
Allowed value for readonly parameter is [true, false] .
If value is true, It will allow you to change the rating.
if value is false, it will not allow you to change the rating.

Currently there is no option to remove reset button i think, in upcoming release it will be available.

**Comment by delongesT** [comment]
Yes it is possible to remove the reset button, but the tweak is not in the JS, but HTML.
Edit the file named **rating-stars-directive.html** and remove the Anchor link which has **remove_circle_outline** as the material icon.

![image](https://user-images.githubusercontent.com/29718649/45450872-0f9e2580-b6f7-11e8-996a-d7f3a2e7c113.png)


> 

>
